cli/tests/pcluster/templates/test_cluster_stack/test_efs_shared_storage_encryption/config-encrypted.yaml

```diff
@@ -0,0 +1,25 @@
+Image:
+  Os: alinux2
+HeadNode:
+  InstanceType: t3.micro
+  Ssh:
+    KeyName: ec2-key-name
+  Networking:
+    SubnetId: subnet-12345678
+  SharedStorageType: Efs
+  SharedStorageEfsSettings:
+    Encrypted: true
```

Comment:
[Non blocking comment] [Knowledge Sharing] You can also inject variables into the config from the test to avoid duplication of config files and make it more explicit what you're chagning in it. In this case you would inject the encrypted variable.